jgreen44/assign1git
# chore(assign1): Add comments to README.md

Addresses: Task 2 - Assignment 1

**Summary**

This PR added comment lines to the README file for assignment 1. 

**Checklist**

- The basics
  - [x] I tested these changes manually in my local or dev environment
- Tests
  - [ ] Added or updated
- Event Tracking
  - [ ] I added event tracking and followed the event tracking guidelines
- Error Reporting
  - [ ] I reported errors and followed the error reporting guidelines

**Steps To Test**

Add steps on how to test this PR.

1. Checkout newbranch
2. View README to make sure comments have been added


**Screenshots**

n/a

**Additional context**

n/a
